pkg/collector/corechecks/containerimage/processor.go

```diff
@@ -52,13 +53,27 @@ func (p *processor) processRefresh(allImages []*workloadmeta.ContainerImageMetad
 }
 
 func (p *processor) processImage(img *workloadmeta.ContainerImageMetadata) {
+	var lastCreated *timestamppb.Timestamp = nil
 	layers := make([]*model.ContainerImage_ContainerImageLayer, 0, len(img.Layers))
 	for _, layer := range img.Layers {
+		var created *timestamppb.Timestamp = nil
```

Comment:
It would be good to move this logic to calculate `lastCreated` to workloadmeta if in the future other checks need this attribute too, but for now I think it's OK :+1: